src/libraries/System.CodeDom/tests/System/CodeDom/Compiler/CodeCompilerTests.cs

```diff
@@ -458,6 +458,7 @@ public void FromSourceBatch_NullSources_ThrowsArgumentNullException()
         [InlineData(null)]
         [InlineData("")]
         [InlineData("cmdArgs")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
Should we add a Reason explaining why it's being skipped?

Reply:
Sure